File: Policlinica.Pacient.Angular/ClientApp/src/app/models/pacient.ts
import { LoginPacient } from './login.pacient';

export class Pacient{
    PacientId:number;
    Prenume:string;
    Nume:string;
    CNP:string;
    Email:string;
    CardSanatate:string;
    Ocupatie:string;
   // LoginPacient:LoginPacient;
    constructor(firstName:string,lastName:string,cnp:string,email:string,cnas:string,job:string){
        this.Prenume=firstName;
        this.Nume=lastName;
        this.CNP=cnp;
        this.CardSanatate=cnas;
        this.Email=email;
        this.Ocupatie=job;
        
    }
}

File: Policlinica.Pacient.Angular/Models/Pacient.cs
using System;
using System.Collections.Generic;

namespace Policlinica.Pacient.Angular.Models
{
    public partial class Pacient
    {
        public Pacient()
        {
            ProgramariServicii = new HashSet<ProgramariServicii>();
        }

        public int PacientId { get; set; }
        public string Nume { get; set; }
        public string Prenume { get; set; }
        public string Cnp { get; set; }
        public string CardSanatate { get; set; }
        public string Ocupatie { get; set; }
        public string Email { get; set; }

        public LoginPacient LoginPacient { get; set; }
        public ICollection<ProgramariServicii> ProgramariServicii { get; set; }
    }
}
